src/Compilers/CSharp/Portable/FlowAnalysis/LocalDataFlowPass.cs

```diff
@@ -162,7 +162,7 @@ private int DescendThroughTupleRestFields(ref Symbol symbol, int containingSlot,
                 // force corresponding slots if do not exist
                 while (!TypeSymbol.Equals(containingType, symbol.ContainingType, TypeCompareKind.ConsiderEverything))
                 {
-                    var restField = containingType.GetMembers(NamedTypeSymbol.ValueTupleRestFieldName).FirstOrDefault() as FieldSymbol;
+                    var restField = containingType.GetMembers(NamedTypeSymbol.ValueTupleRestFieldName).FirstOrDefault(s => s is not TupleVirtualElementFieldSymbol) as FieldSymbol;
```

Comment:
:memo: This change is for `TupleWithReservedMemberNames` (we want to pick up the correct `Rest` field, but the order changed a bit)